lib/beaker-hostgenerator/util.rb

```diff
@@ -22,7 +22,7 @@ def pe_dir(version, family)
 
     def fixup_node(cfg)
       # PE 2.8 doesn't exist for EL 4. We use 2.0 instead.
-      if cfg['platform'] =~ /el-4/ and PE_VERSION =~ /2\.8/
+      if cfg['platform'] =~ /el-4/ and BeakerHostGenerator::Data.pe_version =~ /2\.8/
```

Comment:
Yes, but changing it will require a major version bump. (It is supported here for backwards compatibility with https://github.com/puppetlabs/sqa-utils-gem/blob/master/bin/genconfig#L259-L264)